test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/PushCommandTest.cs

```diff
@@ -55,8 +62,8 @@ public void PushCommand_AllowsTimeoutToBeSpecifiedHigherThan100Seconds()
                     // Assert
                     server.Stop();
                     Assert.True(0 == result.Item1, $"{result.Item2} {result.Item3}");
```

Comment:
nit: You can group the Assert.True(0 == resutl1.Item1) and Assert.Contains(MESSAGE_PACKAGGE..., result.Item2) with Util.VerifyResultSucess();

Reply:
Good suggestion, but this isn't a test I modified. I did replicate it on my tests, but don't see much value in refactoring this or touching other tests.